source/DomainTheory/Lifting/LiftingSet.lagda

```diff
@@ -171,6 +171,54 @@ module _ {𝓤 : Universe}
 
 \end{code}
 
+Minor addition by Ayberk Tosun.
```

Comment:
Can you please write a comment saying something along the lines of "The lifting of a set as a dcpo as defined above has an order that is essentially locally small. For some purposes it is convenient to repackage the lifitng with the equivalent order that has small values."

Reply:
Comment added.